tests/ui/const-generics/generic_const_parameter_types/check-type-in-mir.stderr

```diff
@@ -1,27 +1,19 @@
-warning: the feature `unsized_const_params` is incomplete and may not be safe to use and/or cause compiler crashes
```

Comment:
ah hmm :(

I kinda forgot that switching to a library feature would mean we no longer consider feature(unsized_const_params) to be incomplete :thinking: Do you think you could re-add `unsized_const_params` as a langauge feature just so that we continue getting these warnings? It shouldn't actually be used anywhere in the compiler, I just want people to get pushed away from actually using this for now lol